### DISCUSSION TITLE
What is the name of a smooth and creamy seasoned soup from 🇫🇷?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#679] [Galaxy Trivia] [answered]
**Hint:** Typically based on a strained broth of crustaceans.

**Bonus:** List the main ingredients by using emojis.

**Comment by peterbaccaro** [comment]
**Bisque** is the smooth, creamy, highly seasoned soup of French origin, classically based on a strained broth (coulis) of crustaceans.

🦞 🦀 🦐  🧈  🧅 🧄 🍅 💧

**Comment by SANKARAMDAS** [comment]
Bisque is a smooth, creamy, highly seasoned soup of French origin, classically based on a strained broth (coulis) of crustaceans.

🦞🦀🦐🍅🍎🧈 🧅 🧄

**Comment by ParanoidUser** [comment]
@MarieCamillePetit / @DevElkami / @SebastienXia, we need your honest opinion on this case. As an expert committee, please cast your vote for the best answer. 🤔⚖️

**Comment by nsafarova** [comment]
The smooth and creamy seasoned soup from 🇫🇷 France that is typically based on a strained broth of crustaceans is called "bisque."

Here are the main ingredients of bisque represented by emojis:

🦞 - lobster (or other crustaceans such as crab or shrimp)
🍅 - tomato
🧅 - onion
🥕 - carrot
🧄 - garlic
🥛 - cream (or milk)
🍚 - rice (sometimes used as a thickener)
🧈 - butter
🌿 - herbs (such as thyme, bay leaves, and parsley)
🧂 - salt (and other seasonings)

**Comment by thuongtruong1009** [comment, marked as answer]
The name of a smooth and creamy seasoned soup from France is "vichyssoise".

🥔🧅🍶➕🥛 = Vichyssoise

The main ingredients are:

Potatoes 🥔
Onions or leeks 🧅
Cream 🍶
Milk 🥛
